eventmesh-common/src/main/java/org/apache/eventmesh/common/Constants.java

```diff
@@ -28,6 +28,8 @@ public class Constants {
 
     public static final String DATE_FORMAT_DEFAULT = "yyyy-MM-dd HH:mm:ss";
 
+    public static final String DATA_CONTENT_TYPE = "DataContentType";
```

Comment:
the DataContentType key in the CloudEvents protocol must be lowercase，please refer to https://github.com/cloudevents/spec/blob/v1.0.2/cloudevents/spec.md

Reply:
resolved.